corehq/blobs/mixin.py

```diff
@@ -279,6 +283,10 @@ def get_external_blobs(doc, default=None):
     def blobs(self):
         return BlobMixin.blobs.fget(self)
 
+    @property
+    def blob_identifier(self):
+        return self._id
```

Comment:
BlobHelper seems to be never used for xform-errors, so I didn't have to override it, does this sound correct? @snopoke